site/en/docs/extensions/mv3/known-issues/index.md

```diff
@@ -20,17 +20,23 @@ When a capability is added to the platform or a bug is addressed, it will be rem
 
 This section lists major features that will be added to the Manifest V3 platform in order to aid developers migrating from Manifest V2. Timelines included here are best effort estimates, not hard commitments.
 
-### webRequest.onAuthRequired events {: #webrequest-onauthrequired }
+### FIXED: webRequest.onAuthRequired events {: #webrequest-onauthrequired }
 
-**Summary:** User-installed Manifest V3 extensions cannot currently intercept `webRequest.onAuthRequired` events, since the `webRequestBlocking` permission is restricted to policy-installed extensions. Chrome provides extensions with a way to supply credentials for authentication requests in Manifest V3.
+**Summary:** User-installed Manifest V3 extensions could not currently intercept `webRequest.onAuthRequired` events because the `webRequestBlocking` permission is restricted to policy-installed extensions. Chrome now allows extensions with the `webRequest` and `webRequestAuthProvider` permissions to asynchronously supply credentials for `onAuthRequired` events.
```

Comment:
```suggestion
**Summary:** User-installed Manifest V3 extensions could not intercept `webRequest.onAuthRequired` events because the `webRequestBlocking` permission is restricted to policy-installed extensions. Chrome now allows extensions with the `webRequest` and `webRequestAuthProvider` permissions to asynchronously supply credentials for `onAuthRequired` events.
```